one-shot-device/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+	const app = await NestFactory.create(AppModule);
+	await app.listen(process.env.PORT || 8080);
+	console.log(`Integration Service One Shot Device Registration running on: ${await app.getUrl()}`);
```

Comment:
please use a logger like it is done here: https://github.com/iotaledger/cxc-slu-authenticity/blob/main/authenticity-collector/src/identity/identity.service.ts#L13